air/src/trace/main_trace.rs

```diff
@@ -207,12 +258,12 @@ impl MainTrace {
         let col_b6 = self.columns.get_column(DECODER_TRACE_OFFSET + 7);
         let [b0, b1, b2, b3, b4, b5, b6] =
             [col_b0[i], col_b1[i], col_b2[i], col_b3[i], col_b4[i], col_b5[i], col_b6[i]];
-        b0 + b1.mul_small(2)
-            + b2.mul_small(4)
-            + b3.mul_small(8)
-            + b4.mul_small(16)
-            + b5.mul_small(32)
-            + b6.mul_small(64)
+        b0 + b1 * Felt::from_u64(2)
+            + b2 * Felt::from_u64(4)
+            + b3 * Felt::from_u64(8)
+            + b4 * Felt::from_u64(16)
+            + b5 * Felt::from_u64(32)
+            + b6 * Felt::from_u64(64)
```

Comment:
nit: given that these are small values, could using `Felt::from_u16()` be more efficient?

Reply:
The from methods for Goldilocks are all the same and free, since the internal representation is natural and not necessarily reduced.

I can't check now, but if the underlying method does bounds checking, then it might be that `from_u{8,16,32}` are slightly faster since no bound checks would happen.

Reply:
I think, for Goldilocks, all of these boil down to a call to `Felt::new()`. If we were in the situation where the internal value was always reduced modulo the prime or there were faster reduction methods for smaller values then `Felt::from_u16()` could be meaningfully different from `Felt::new()`.

The question then is, should we keep these just to be future proof, or should we simplify our codebase by always using `Felt::new()` ?

Reply:
I think new is probably easier, and we can remove a lot of the `n_uX` suffixes when inputting literals. This should be something we can easily refactor with `comby`.